src/invisible/how-tos/data-collection-with-the-companion-app/achieve-super-precise-time-sync.md

```diff
@@ -55,4 +56,4 @@ device.close()
 
 Using continuous offset estimates like this, you can precisely compensate for clock drifts by correcting the respective timestamps.
 
-When using a wifi connection, the speed of data transfer can vary at times, leading to inaccuracies in the estimation. If precise synchronization down to a few milliseconds is required, it is recommended to use an ethernet connection. You can connect an ethernet cable to the phone using an appropriate USB-C adapter or hub.
\ No newline at end of file
+When using a wifi connection, the speed of data transfer can vary at times, leading to inaccuracies in the estimation. If precise synchronization down to a few milliseconds is required, it is recommended to use an ethernet connection. You can connect an ethernet cable to the phone using an appropriate USB-C adapter or hub.
```

Comment:
@marc-tonsen maybe it is worth specifying the hub is only available for One Plus 8 and 8t?

Reply:
Agreed! I wonder if instead of putting this in here, we should link out to a dedicated page on using HUBs. Maybe as part of the "Glasses & Companion Device" section?

Reply:
makes sense, as they might use it for charging only, and not for the ethernet capabilities

Reply:
I have added an according section and linked to it. Please check it out!